docs/CAYENNEAPI.md

```diff
@@ -372,339 +54,38 @@ In addition to oAuth2, our authentication service also incorporates Single Sign
 | 400  |  Bad Request: payload validation     |
 | 401  |  Unauthorized: Invalid or expired token used.     |
 
-All authentication requests are directed to **``https://auth.mydevices.com``**.
-
-#### User Management
-
-This authentication service also provides routes for user creation, modification and information.
-All requests need to include a Authorization header and API version, for example:
-
-| Header	| Value  |
-|-------|-------------------------------------------|
-| Authorization  |  Bearer JWT_TOKEN |
-| X-API-Version  |  1.0     |
+All authentication requests are directed to **``https://accounts.mydevices.com``**.
 
 ##### Create User
 
-<table style="width: 100%;">
-<tbody>
-<tr>
-<td style="font-size: 15px; padding: 10px; background-color: #79ae3d; color: #ffffff;"><b>POST</b></td>
-<td style="font-size: 15px; padding: 10px; background-color: #d9ead3; color: #000000;">/users</td>
-</tr>
-</tbody>
-</table>
-
-**Parameters**
-
-| Parameter	| Description |	Type |
-|--------------------|--------------------|---|
-| first_name \*      | first name of user | String |
-| last_name \*       | last name of user | String |
-| email \*           | user’s email | String |
-| password \*        | user's password | String |
-| user_class_id      | identify user class | Integer |
-| user_group_id      | categorize user group | Integer |
-| application_id     | adds a user to an application | String |
-| permission_role_id | permission role associated with user | Integer |
-
-\* *required*
-
-**Response Model**
-
-```
-{
-  “id”: UUID,
-  "first_name": "",
-  "last_name": "",
-  "email": "",
-  "password": "",
-  "user_class_id": 0,
-  "permission_role_id": 0,
-  "user_groups_id": 0,
-  "application_id": “"
-}
-```
-
-##### Get User
+With OAuth the user creation is made directly on the identity provider web page. The url to directly acess the user registration page is **``https://accounts.mydevices.com/auth/realms/cayenne/login-actions/registration``**. Refer to the Application section and the different OAuth flows for more details and the paramters list that need to passed.
```

Comment:
With OAuth the user creation is made directly on the identity provider web page. The url to directly acess the user registration page is **``https://accounts.mydevices.com/auth/realms/cayenne/login-actions/registration``**. Refer to the Application section and the different OAuth flows for more details and the paramters list that need to be passed.